setup.py

```diff
@@ -16,7 +16,7 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    'html5lib>=0.99999999,>=1.0b9',
```

Comment:
How does this work? Seems like the second `>=` overrides the first since a version number can't satisfy both clauses.